frontend/packages/shared/src/hooks/useUniqueKey.test.ts

```diff
@@ -0,0 +1,32 @@
+import { act, renderHook } from '@testing-library/react';
+import { type UseUniqueKeyArgs, useUniqueKey } from 'app-shared/hooks/useUniqueKey';
+
+describe('useUniqueKey', () => {
+  it('should generate unique keys within a max number range', () => {
+    const { result } = renderHook(() => useUniqueKey({ maxNumberOfKeys: 2 }));
+
+    const firstItem = result.current.getUniqueKey(0);
+    const secondItem = result.current.getUniqueKey(1);
+
+    expect(result.current.getUniqueKey(0)).toEqual(firstItem);
+    expect(result.current.getUniqueKey(1)).toEqual(secondItem);
+    expect(result.current.getUniqueKey(1)).not.toEqual(firstItem);
+  });
```

Comment:
For å sjekke at det genereres nøyaktig to elementer, bør vi også verifisere at det ikke er noe på indeks `2`.
```
expect(result.current.getUniqueKey(2)).toBeUndefined();
```

Reply:
Fixed! 😊